### PR TITLE
https://issues.redhat.com/browse/ACM-11429 Z-stream errata for MCE 2.5.3

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Review by rhaignere — APPROVED** [review]
LGTM, thank you!

File: clusters/release_notes/errata.adoc [hunk 1]
```diff
@@ -5,6 +5,12 @@ For {mce-short}, the Errata updates are automatically applied when released.
 
 *Important:* For reference, link:https://access.redhat.com/errata/#/[Errata] links and GitHub numbers might be added to the content and used internally. Links that require access might not be available for the user. 
 
+== Errata 2.5.3
+
+* Added a field in the KubeVirt creation wizard that sets the default mode for the Hosted Control Plane cluster to `HighAvailability` mode. (link:https://issues.redhat.com/browse/ACM-10580[ACM-10580])
```
Comment (dockerymick) [inline clusters/release_notes/errata.adoc]: Throughout our doc, i see that we use sentence case for hosted control plane.
```suggestion
* Added a field in the KubeVirt creation wizard that sets the default mode for the hosted control plane cluster to `HighAvailability` mode. (link:https://issues.redhat.com/browse/ACM-10580[ACM-10580]).
```

**Review by dockerymick — APPROVED** [review]
Made a suggestion

**Review by rhaignere — APPROVED** [review]
Thanks for the update, lgtm

File: clusters/release_notes/errata.adoc [hunk 2]
```diff
@@ -5,6 +5,12 @@ For {mce-short}, the Errata updates are automatically applied when released.
 
 *Important:* For reference, link:https://access.redhat.com/errata/#/[Errata] links and GitHub numbers might be added to the content and used internally. Links that require access might not be available for the user. 
 
+== Errata 2.5.3
+
+* Added a field in the KubeVirt creation wizard that sets the default mode for the hosted control plane cluster to `HighAvailability` mode. (link:https://issues.redhat.com/browse/ACM-10580[ACM-10580]).
```
Comment (dockerymick) [inline clusters/release_notes/errata.adoc]: Sorry for adding the extra period. I realize you are placing the issue in parentheses 
```suggestion
* Added a field in the KubeVirt creation wizard that sets the default mode for the hosted control plane cluster to `HighAvailability` mode. (link:https://issues.redhat.com/browse/ACM-10580[ACM-10580])
```

**Review by dockerymick — APPROVED** [review]

**Pull request merged**